release/release_utils.py

```diff
@@ -977,14 +991,6 @@ def mvn_sign_and_deploy_all(url, repo_id, pom_file, artifact_jar, source_jar, ja
 # Misc. Utils
 
 def update_htaccess( releaseDir, cfLtVersion, afuVersion, htAccessTemplate, htAccessToReplace, auto ):
```

Comment:
Most of the parameters to this function are unused.  Is this even necessary any longer?  It's now a one-liner that could be substituted at its single invocation location.